frontend/packages/topology/src/components/utils/checkResourceQuota.tsx

```diff
@@ -0,0 +1,37 @@
+import { getUsedPercentage } from '@console/app/src/components/resource-quota/utils';
+import { ResourceQuotaModel } from '../../../../../public/models';
+
+export const checkQuotaLimit = (resourecQuota: any) => {
+  let quotaName = '';
+  let quotaKind = '';
+  const resourceQuotaResources = resourecQuota.map((quota) => {
```

Comment:
will `resourecQuota` always be there else will be good to add a check here

Reply:
`resourecQuota` if no RQ also it will be empty array so no need of extra check here.